orchagent/copporch.cpp

```diff
@@ -94,10 +152,27 @@ void CoppOrch::getTrapIdList(vector<string> &trap_id_name_list, vector<sai_hosti
     }
 }
 
+bool CoppOrch::applyAttributesToTrapIds(vector<sai_hostif_trap_id_t> &trap_id_list, std::vector<sai_attribute_t> &trap_id_attribs)
```

Comment:
here you don't have std:: before the first vector and have std:: before the second vector. Good to remove all std:: in .cpp file for cleaning up.